packages/react-tabster/src/hooks/useKeyborg.ts

```diff
@@ -0,0 +1,35 @@
+import { createKeyborg } from 'keyborg';
+import { KeyborgCallback } from 'keyborg/dist/Keyborg';
+import { RefObject, useEffect, useRef } from 'react';
+import { KEYBOARD_NAV_ATTRIBUTE } from '../symbols';
+import { useConst } from '@fluentui/react-utilities';
+
+/**
+ * instantiates keyborg and add attribute to ensure focus indicator synced to keyborg logic
+ */
+export function useKeyborg<E extends HTMLElement>() {
+  const keyborg = useConst(() => createKeyborg(window));
```

Comment:
```suggestion
  const { targetDocument } = useFluent()
  const keyborg = useMemo(() => createKeyborg(targetDocument.defaultView), [targetDocument]);
```

We use a document/window from a context and handle its updates.